Libraries/Components/WebView/WebView.ios.js

```diff
@@ -503,11 +505,13 @@ class WebView extends React.Component {
       this.props.decelerationRate,
     );
 
-    const source = this.props.source || {};
-    if (this.props.html) {
-      source.html = this.props.html;
-    } else if (this.props.url) {
-      source.uri = this.props.url;
+    const source: Object | number = this.props.source || {};
```

Comment:
How come source can be a number?

Reply:
It comes from this line. `source` supposed to be `number` in some situations.
I don't find the situation invoked with `number` but I followed previous types.

https://github.com/facebook/react-native/blob/master/Libraries/Components/WebView/WebView.ios.js#L165